en/debug/gdb_debugging.md

```diff
@@ -69,10 +69,25 @@ The numbers in the graph will all be zero, because they are in MB. Just look at
 
 See the [gperftools docs](https://htmlpreview.github.io/?https://github.com/gperftools/gperftools/blob/master/docs/heapprofile.html) for more information.
 
+## Hard Fault Debugging
 
-## Debugging Hard Faults in NuttX
+A hard fault is a state when the operating system detects that it has no valid instructions to execute.
+This is typically the case when key areas in RAM have been corrupted.
 
-A hard fault is a state when the operating system detects that it has no valid instructions to execute. This is typically the case when key areas in RAM have been corrupted. A typical scenario is when incorrect memory access smashed the stack and the processor sees that the address in memory is not a valid address for the microprocessors's RAM.
+
+### Video
```

Comment:
FYI, this is just a "quick and dirty" way to get the video on hard fault debugging into docs. This section could certainly be revisited.